oat-sa/extension-tao-itemqti
# Fix/pisa25 472/id attribute removed

related to https://oat-sa.atlassian.net/browse/PISA25-472

### Description

This PR adds comments to generated code to warn author if he uses sourceDialog to edit code of the interaction

### How to test

Can be validated on kitchen env

Create item with widget, i.e. img (block or inline). Check the source code - the widget code should be wrapped with warning comments 

``
widget code
``

when item is saved, saveItem should not contain these comments
